packages/suite/src/views/suite/device-recovery-mode/index.tsx

```diff
@@ -11,42 +11,49 @@ const Wrapper = styled.div`
     flex-direction: column;
 `;
 
-const Buttons = styled.div`
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
-    margin: 20px;
+// const Buttons = styled.div`
+//     display: flex;
+//     flex-direction: row;
+//     justify-content: center;
+//     margin: 20px;
+// `;
+
+const StyledImage = styled(Image)`
+    flex: 1;
 `;
 
-const Index = ({ recovery, device, checkSeed, recoverDevice, goToStep, addPath }: Props) => (
-    <Wrapper>
-        {recovery.status === 'in-progress' && <Loading />}
-        {recovery.status === 'initial' && (
-            <>
-                <H2>
-                    <Translation id="TR_DEVICE_IN_RECOVERY_MODE" />
-                </H2>
-                <Buttons>
-                    {!device?.features?.initialized && (
-                        <Button
-                            onClick={() => {
-                                recoverDevice();
-                                goToStep('recovery');
-                                addPath('recovery');
-                            }}
-                        >
-                            <Translation id="TR_CONTINUE" />
-                        </Button>
-                    )}
-                    {device?.features?.initialized && (
-                        <Button onClick={() => checkSeed()}>
-                            <Translation id="TR_CONTINUE" />
-                        </Button>
-                    )}
-                </Buttons>
-            </>
-        )}
-    </Wrapper>
-);
+const Index = ({ device }: Props) => {
```

Comment:
since I am now handling recovery-mode automatically, it probably makes sense to replace this component with simple <Loading />